README.md

```diff
@@ -21,7 +21,14 @@
 
 # What is NebulaGraph?
 
-**NebulaGraph** is an open-source graph database capable of hosting super large scale graphs with dozens of billions of vertices (nodes) and trillions of edges, with milliseconds of latency.
+**NebulaGraph** is an open-source graph database capable of hosting super large scale graphs <br/>
+with dozens of billions of vertices (nodes) and trillions of edges, with milliseconds of latency. 
+
+
+It has a lot of users, including some famous and big companies of **BAT**, **TMD**,  <br/>
```

Comment:
The term BAT and TMD isn't a thing to English users, we may reword it here.

Reply:
Any better words? We cannot directly use 'Tencent', 'Meituan' on GitHub yet😂